sdcm/nemesis.py

```diff
@@ -3457,9 +3457,19 @@ def decommission_post_action():
         if self.cluster.params.get('cluster_backend') == 'azure':
             timeout += 1200  # Azure reboot can take up to 20min to initiate
         ParallelObject(objects=[trigger, watcher], timeout=timeout).call_objects()
+
+        def wait_restarted_node_fully_start(node):
+            self.log.info('Waiting scylla services to start after node reboot')
+            node.wait_db_up()
+            self.log.info('Waiting JMX services to start after node reboot')
+            node.wait_jmx_up()
+            self.cluster.wait_for_nodes_up_and_normal(nodes=[node])
```

Comment:
Doing all three checks  seems like an overkill to me...

Reply:
in one hand, if we moved to the next check, it should no take much... we could have only the 3rd one, but maybe it is not enough?